### PR TITLE
[Chore] Move temp db format into a helper.

### PR DESCRIPTION
**Pull request by ruseinov**
## Summary of changes

Changes introduced in this pull request:

- Moved temporary_db_name format into a helper. 

## Reference issue to close (if applicable)

Closes

## Change checklist

- [x] I have performed a self-review of my own code,
- [x] I have made corresponding changes to the documentation,
- [x] I have added tests that prove my fix is effective or that my feature works (if possible),
- [x] I have made sure the [CHANGELOG][1] is up-to-date. All user-facing changes should be reflected in this document.

[1]: https://github.com/ChainSafe/forest/blob/main/CHANGELOG.md

### CODE REVIEW

**Review by LesnyRumcajs — CHANGES_REQUESTED** [review]
There's one more entry. https://github.com/ChainSafe/forest/blob/main/src/db/migration/migration_map.rs#L445

File: src/db/migration/db_migration.rs [hunk 1]
```diff
@@ -79,6 +80,10 @@ impl DbMigration {
     }
 }
 
+pub(crate) fn db_name(from: &Version, to: &Version) -> String {
```
Comment (LesnyRumcajs) [inline src/db/migration/db_migration.rs:83]: Plus, I think it belongs more in the `migration_map.rs` file.

Comment (ruseinov) [reply src/db/migration/db_migration.rs:83]: Since it's used to implement `db_migration` trait's method - I don't see a reason to move it out of the file. But if needed - we can do it. No strong opinion here.

Comment (LesnyRumcajs) [reply src/db/migration/db_migration.rs]: It's used to implement `MigrationOperation` trait methods defined in `migration_map.rs`, no?

File: src/db/migration/migration_map.rs [hunk 2]
```diff
@@ -470,3 +471,7 @@ mod tests {
         assert!(migration.post_checks(temp_dir.path()).is_ok());
     }
 }
+
+pub(crate) fn db_name(from: &Version, to: &Version) -> String {
```
Comment (LesnyRumcajs) [inline src/db/migration/migration_map.rs:475]: Do we still need it in the trait? Let's simplify things and just have a free `temporary_db_name` function. I guess it _may_ be overridden in the future, but probably YAGNI.

Comment (ruseinov) [reply src/db/migration/migration_map.rs:475]: It is not in the trait.

Comment (LesnyRumcajs) [reply src/db/migration/migration_map.rs:475]: Let me rephrase. `temporary_db_name` _is_ in the trait, and it's just a proxy for `db_name`. We could use the `db_name` directly (though renamed to be explicit for the purpose). Less code, less bugs. What do you think?

Comment (ruseinov) [reply src/db/migration/migration_map.rs:475]: Yeah, that makes sense. I'll incorporate that.

Comment (ruseinov) [reply src/db/migration/migration_map.rs:475]: I just realised that `temporary_db_name` is never used outside of any given migration, so there is no reason to have it as a trait member in the first place. This has been addressed.

**Review by LesnyRumcajs — APPROVED** [review]

**Review by lemmih — APPROVED** [review]
